base.go

```diff
@@ -46,28 +46,26 @@ func (b Base) Validate() error {
 func (b Base) String() string {
 	str := strings.ToLower(b.Name)
 	if !b.Channel.Empty() {
-		str += "/" + b.Channel.String()
+		str += "@" + b.Channel.String()
```

Comment:
Turns out, this is going to be REALLY painful for the parsing logic. For example, if you have the following:

```go
base, err := ParseBase(Base{Name: "ubuntu"}.String())
// err == NotValid
```